mobile/pubspec.lock

```diff
@@ -508,10 +508,10 @@ packages:
     dependency: "direct main"
     description:
       name: meta
-      sha256: "12307e7f0605ce3da64cf0db90e5fcab0869f3ca03f76be6bb2991ce0a55e82b"
+      sha256: "6c268b42ed578a53088d834796959e4a1814b5e9e164f147f580a386e5decf42"
       url: "https://pub.dev"
     source: hosted
-    version: "1.9.0"
+    version: "1.8.0"
```

Comment:
all these downgrades/upgrades are a bit unfortunate..... they don't align if you run `flutter pub get`?

Reply:
This is indeed weird, because there was no change to the `pubspec.yml` - why do we have changes in the `pubspec.lock` file? 😅

I reckon this could be because somebody a different PR did not update the `pubspec.lock` correctly and merged - I think our CI action might not cover that properly. I created: https://github.com/get10101/10101/issues/230